views/users.ejs

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Users</title>
+
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
I'd suggest running this file through prettier, as `<meta>` doesn't have the Netscape 4 space, but the other `<link>` tags do